temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
```

Comment:
Handle exceptions when opening the file using a try-except block. This will prevent the program from crashing if the file does not exist or cannot be opened. [important]